### ISSUE TITLE
Snoozy overlaps the top bar at short window sizes

### ISSUE THREAD

**Issue by fwextensions** [deanoemcke/thegreatsuspender#750]
![image](https://user-images.githubusercontent.com/61631/45125608-20153580-b124-11e8-8954-5cc1d9a30c5a.png)

### Specifications:
 * Extension version:  7.0.110
 * Browser & version: Chrome 68
 * Operating system & version: Win10

**Comment by liamjohnston** [comment]
Thanks @fwextensions 

Out of interest, is this a real browser size you've been using? Or just seeing what would happen to the suspended tab at small resolutions?

Looking at the browser sizes in our analytics, I'm seeing that _practically_ no one browses at the <~450px height required for this bug to occur. 

So probably labelling this as a won't fix unless some more pressure mounts up :D

**Comment by liamjohnston** [comment]
Ah, I just spied your PR for this.
https://github.com/deanoemcke/thegreatsuspender/pull/751

Thanks for the fix. I'm still hesitant about this - it's adding extra 10 lines of code (plus more to fix the spinner placement issue). After the fallout from the initial 7.x release CSS performance issues, I'm keen to avoid too much unneeded code. 

@deanoemcke thoughts...?

**Comment by fwextensions** [comment]
Yeah, this isn't a size a window usually stays at, but I've noticed it when detaching external monitors, or moving windows between monitors with different UI scaling.  You can end up with weirdly sized windows, and it seemed like it would look cleaner without any overlap.  

I'd say at least stick a `z-index` on the top bar so the text isn't obscured.

![image](https://user-images.githubusercontent.com/61631/45126797-0e369100-b12a-11e8-8d71-bfc44f18c248.png)

**Comment by deanoemcke** [comment]
If it can be fixed purely with css, without any extra code to handle the spinner then I'm all for it.
I would rather head in the direction of simplicity if possible.

**Comment by liamjohnston** [comment]
The z-index one-liner seems the best solution to me, in terms of helping a very uncommon situation with minimal performance impact or testing required:
`z-index: 101;` (I know) on `.gsTopBar` does the job.

**Comment by fwextensions** [comment]
Sounds good.  I'll close the PR.  It was mostly an excuse to refresh my memory on how flexbox works.
